draft a blog post about email-compat (#439)
deltachat/deltachat-pages: _posts/2021-05-05-email-compat.md

---
title: E-mail compatibility releases
author: bjoern
image: ../assets/blog/2021-05-html-mail.jpg
---

The current releases make
interoperability with classic e-mail land even smoother,
targeting several long-awaited things.

## Rich-formatted HTML-mails

_Got a train ticket?_
_A notification from a parcel service?_
_An invitation?_

<img src="../assets/blog/2021-05-html-mail.jpg" width="250" style="float: right; clear:both; margin-left:.1em; margin-bottom:.2em;" alt="Screenshot with &quot;Show classic e-mails&quot; option" />
All these messages may come as **rich-formatted HTML-mails**,
that are now displayed nicely directly inside Delta Chat 1.20.

To access them,
just use the **Show full message** button
that will be added just to the bubble whenever needed.

## Mailing lists

Delta Chat 1.20 supports **reading mailing lists.**

Mailing lists are especially important as
also "normal" mails may come as mailing lists -
e.g. the mentioned notification from the parcel service
is now displayed even if wrapped into a mailing list.

As sender on mailing lists may be anonymized by the mailing list software,

> they are are marked by a leading tilde as **~Alice**.

Reviewer: ```suggestion
they are marked by a leading tilde as **~Alice**.
```

- [Bots](https://delta.chat/en/2020-03-26-shining-some-light-on-bots) also benefit from several features -
  e.g. they also can anonymize users and can send HTML-mails.

- Due to more senders, we added **more and new colors** for contact avatars

- Due to more messages overall, we **speedup global search**

- you can see the **status** (the footer) of each contact
  in the contact's profile (the feature is available on Android, other systems will follow)

## How it works

Since the first day,
Delta Chat uses the most **diverse and distributed** network ever existing
for transporting chat messages: E-mail.

Delta Chat resists the temptation to build another cage on top of that
and works well together with **other providers and apps;**
doing some good jobs in
separating chat messages from classical e-mails.

So, in general, classic e-mail works since the first day:

- You can **chat with any e-mail address,**
  there is no need that the recipient also uses Delta Chat.
  The recipient just receives a classic e-mail then
  (not only a link as some other apps do).

- To **receive classic e-mails** in addition to chat messages,

>   you need to enable **Settings / Show classic mail.**

Reviewer: ```suggestion
  you need to enable **Settings / Chats and media / Show classic emails.**
```

Reviewer: mmm.... don't know how it is on iOS and Desktop, one more reason to sync menu structure across the different platforms

Author: it is correct for ios, and desktop settings are flat anyway.

Depending on the software the company you are writing to uses,
however, the reply may come from a different address -
in the past, that led to another one-to-one chat, so that you miss the context
and conversations become hard.

The current version fixes this issue and you'll stay in the chat context.

## Other improvements

[Nearly 100 issues were targeted](https://github.com/orgs/deltachat/projects/31) 
to improve email-comaptibility,
some other highlights are:

- Messages **look better** if the recipient does not use Delta Chat -
  the subject is nicer, no unexpected avatar attached

- Better **attachment handling**

- Support of **Outlook.com** (and other providers messing up headers ;) 

- Due to more senders, we added **more and new colors** for contact avatars

- **Check all server-folders** for new messages from time to time

- [Bots](https://delta.chat/en/2020-03-26-shining-some-light-on-bots) also benfit from several features -
  eg. they also can anonymize users and can send HTML-mails.

- **Speedup global search**

- you can see the **status** (the footer) of each contact
  in the contact's profile (the feature is available on Android, other systems will follow)

> ## Is it for anyone?

Reviewer 2: ```suggestion
## Is this for everyone?
```

I think "anyone" has the nuance of "Ist es für irgendwen?" while we want to say "Ist das für alle?" (do we want to say that?)
